app/data_mapper_setup.rb

```diff
@@ -0,0 +1,6 @@
+  env = ENV['RACK_ENV'] || 'development'
+  DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{env}")
+  require './app/models/user'
+  require './app/models/cheep'
+  DataMapper.finalize
+  DataMapper.auto_upgrade!
```

Comment:
Every time the app is racked up there is an upgrade on the database happening and it shouldn't. A rake task can fix this issue and you can run it only when you need it.